server/src/test/java/io/crate/analyze/SelectStatementAnalyzerTest.java

```diff
@@ -2730,8 +2729,7 @@ public void testSubscriptExpressionWithUnknownObjectKeyFromAliasedRelationWithSe
         var analyzed = executor.analyze("select obj_dy['missing_key'] from (select obj_dy from e1) alias");
         assertThat(analyzed.outputs()).hasSize(1);
         assertThat(analyzed.outputs().getFirst())
-            .isVoidReference()
-            .hasName("obj_dy['missing_key']");
+            .isField("obj_dy['missing_key']");
```

Comment:
I think this isn't correct, we should still expect a `VoidReference` here. Why do we need to change the tests at all?

Reply:
If we remove https://github.com/crate/crate/pull/17133/files#diff-19d21b7eccc702d7fc57113c5c6ae548b242cf9d35e80ac1cc51d5c5c0d5c485L119-L123, this test fails. `obj_dy['missing_key']` will become a `ScopedSymbol`.

Reply:
ah right, sry!